src/web/modules/transactions/discrepancies/validateLedger.http.ts

```diff
@@ -1,10 +1,12 @@
 import { Request, Response, NextFunction } from 'express'
-import {diff, DiffEdit} from 'deep-diff'
+import {diff, DiffDeleted, DiffEdit, DiffNew} from 'deep-diff'
 import { Ledger, Connector } from '../../../../lib/pay-request/client'
 import * as _ from 'lodash'
 import moment from 'moment'
 
 const EDIT = 'E'
+const DELETED = 'D'
+const NEW = 'N'
```

Comment:
Could be a [String enum](https://www.typescriptlang.org/docs/handbook/enums.html#string-enums)

Reply:
thanks, done